rest-api/etl/source_to_concept_map.csv

```diff
@@ -24,7 +26,7 @@ BlackSpecific_Liberian|1585618|ppi-race|Liberian|38003598|Race|1970-1-1|2099-12-
 BlackSpecific_Nigerian|1585624|ppi-race|Nigerian|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_Somali|1585619|ppi-race|Somali|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_SouthAfrican|1585620|ppi-race|South African|38003598|Race|1970-1-1|2099-12-31|\N|1
-Hispanic_HispanicSpecific|1586156|ppi-race|Hispanic, Latino, or Spanish (For example: Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.) *|0|\N|\N|\N|\N|2
+Hispanic_HispanicSpecific|1586156|ppi-race|Hispanic, Latino, or Spanish (For example: Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.) *|38003563|Ethnicity|1970-1-1|2099-12-31|\N|2
 HispanicSpecific_Mexican|1585345|ppi-race|Mexican or Mexican American|0|\N|\N|\N|\N|1
```

Comment:
Should HispanicSpecific_Mexican and HispanicSpecific_PuertoRican also have ethnicity set?

Reply:
Right now we don't have a mapping for this. We'll need to introduce them. I'll make that change when we create them.